### PR TITLE
POST request in RegisterServlet

### PR DESCRIPTION
**Pull request by ghost**
So here I handled the POST request in the register page. The RegisterServlet class rejects existing usernames and non-alphanumeric usernames. If the registration is successful it redirects to the login page and adds the new user else it displays an error message.

### CODE REVIEW

**Review by scoheleach — APPROVED** [review]

**Comment by scoheleach** [comment]
it makes sense to me, looks good

File: src/main/java/codeu/controller/RegisterServlet.java [hunk 1]
```diff
@@ -12,10 +17,46 @@
...
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response)
+    throws IOException, ServletException {
+
+    String username = request.getParameter("username");
+    String password = request.getParameter("password");
+
+    if (!username.matches("[\\w*\\s*]*")) {
```
Comment (KevinWorkman) [inline src/main/java/codeu/controller/RegisterServlet.java:46]: One thing to think about: the default restriction was that usernames can only contain letters and numbers. Do you want to keep this restriction? This is completely up to the team and how you want your chat app to behave.

(You don't have to do anything now, just something to think about and talk about with the team.)

Comment (ghost) [reply src/main/java/codeu/controller/RegisterServlet.java:46]: I'll keep this in mind!

**Comment by mgomez03** [comment]
I think this looks good too but I think we should also consider what Kevin is saying. I like numbers and letters, but I think we should add like underscores and such. (but not everything) But yeah, good work!!

**Review by mgomez03 — APPROVED** [review]

**Pull request merged**